### ISSUE TITLE
dev XP builds?

### ISSUE THREAD

**Issue by NS-Clone** [Wohlstand/OPN2BankEditor#124]
реквестируетсо какая нибудь более новая XP совместимая версия
(надеюсь что там так же как и в opl эдиторе убрали asio онли
изза работы которого звук только от одного этого редактора)

**Comment by Wohlstand** [comment]
Приветствую!

Qt 5.7 на XP работает, хоть и официально уже всё, первая версия, которая не взлетит, это Qt 5.8.

Вопрос: а какой у Вас процессор? Тестовые сборки, что я собираю, штатно взлетают на Windows XP и работают. В старых сборках проблема в том, что там включены оптимизации SSE2 и т.п., которые не совместимы с Pentium III и старее. Из-за этого я собрал отдельную 32битную сборку Qt с отключением этих опций, чтобы на Pentium III взлетело. Могу в ближайшее время проверить от себя на железе, и если что не так, проверю скрипты сборки.

**Comment by NS-Clone** [comment]
ну sse4 умя есть :)
тобешь под ХРю должно работать?
и проблему надо искать где то у меня?

dev верисия opl редактора работает (видимо та что xp...w10)
тот opl миди плеер тоже работает

**Comment by NS-Clone** [comment]
ну тобешь наверно проблема на стороне gui-я

**Comment by NS-Clone** [comment]
потом еще проверю на "том же проце" на 7-ке

**Comment by NS-Clone** [comment]
потом еще проверю на "том же проце" на 7-ке

**Comment by Wohlstand** [comment]
> эти работают если пропатчить pe с висты на ХРю
> libADLMIDI-MSVC15-Release-Win32
> libADLMIDI-MSVC17-Release-Win32

Смысла особо в этом нету, лучше собирать из исходников сразу как надо. Говоря про MSVC 2015/2017, чтобы сборки были совместимыми с "Хрюшей", надо явно указывать комплект, чтобы был помечен как XP-совметсимый, а дальше MSVC просто плюёт на XP. Ещё один повод в пользу MinGW-w64, какой я использую для всего добра.

**Comment by Wohlstand** [comment]
Так, я недавно импортировать обновления с OPL3-BE, и также сборки должны собраться с совместимостью на старые процы и на WinXP. Если нет возможности проверить, то проверю я сам, у себя дома. Сейчас я в офисе, не могу.

**Comment by NS-Clone** [comment]
иге вроде фурычит
заметил давно но не мог никак скачать/ответить
гитхуб был "сломан" пол года на mypal-е
когда лазил с хромого
для скачиваний были нужны vpn-ы
в actions никого из артифактов не видно
и только сегодня все звезды сошлись :D

**Comment by Wohlstand** [comment]
> иге вроде фурычит заметил давно но не мог никак скачать/ответить гитхуб был "сломан" пол года на mypal-е когда лазил с хромого для скачиваний были нужны vpn-ы и только сегодня все звезды сошлись :D

У меня есть архив https://builds.wohlsoft.ru/ на сервере в Москве, туда летят все сборки. Никакие туннели на фиг не нужны, чтобы взять свежайшее от туда. Если же конечно Вы в России.

**Comment by NS-Clone** [comment]
не в россиях этот сайтек выглядит вот так :)

<img width="676" height="484" alt="Image" src="https://github.com/user-attachments/assets/d1973941-dff5-478d-a6a1-083440c9141e" />

**Comment by Wohlstand** [comment]
> не в россиях этот сайтек выглядит вот так :)

Я знаю одно место, где без туннеля до российских ресурсов недоступ, у меня много знакомых от туда, также через туннели лезут. Но вот немцы, китайцы, американцы, британцы, и прочие иностранцы лезут на мои сервера свободно.

И, что бы ни произошло, желаю удачи, и чтобы всё было хорошо!
